apps/extension/src/routes/popup/approval/approve-deny.tsx

```diff
@@ -42,8 +42,8 @@ export const ApproveDeny = ({
       </Button>
       {ignore && (
         <Button
-          className='w-[32%] py-2.5 text-sm hover:bg-destructive/90'
-          size='md'
+          className='w-1/2 py-3.5 text-base hover:bg-destructive/90'
+          size='lg'
```

Comment:
fix dimensions of site settings button

----------------

_before_

<img width="512" alt="Screenshot 2024-11-24 at 9 48 10 PM" src="https://github.com/user-attachments/assets/2d90bebf-894d-4e89-a79b-ad264e277136">

----------------

_after_

<img width="512" alt="Screenshot 2024-11-24 at 9 51 31 PM" src="https://github.com/user-attachments/assets/e3c649b6-b659-4575-ac5e-083c05879cac">